client/dist/0.4e18c8f5b7e6e004723b.hot-update.js

```diff
@@ -0,0 +1,178 @@
+webpackHotUpdate(0,{
```

Comment:
Hey I think this file is from using webpack --watch. We shouldn't include it in the repo. I think webpack-dev-server doesn't produce these.